### PR TITLE
Test fallback screensharing

### PR DESCRIPTION
**Pull request by dbkr**
## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: spec/unit/webrtc/call.spec.ts [hunk 1]
```diff
@@ -955,8 +955,42 @@ describe('Call', function() {
...
 
             expect(
-                call.getLocalFeeds().filter(f => f.purpose == SDPStreamMetadataPurpose.Screenshare).length,
-            ).toEqual(0);
+                call.getLocalFeeds().filter(f => f.purpose == SDPStreamMetadataPurpose.Screenshare),
+            ).toHaveLength(0);
         });
     });
+
+    it("falls back to replaceTrack for opponents that don't support stream metadata", async () => {
```
Comment (SimonBrandner) [inline spec/unit/webrtc/call.spec.ts:963]: Could we test `replaceTrack()` was called?

**Review by SimonBrandner — COMMENTED** [review]
Otherwise LGTM

File: spec/unit/webrtc/call.spec.ts (hunk 1 above)
Comment (dbkr) [reply spec/unit/webrtc/call.spec.ts:964]: Yeah, with a bit of re-arranging

File: spec/test-utils/webrtc.ts [hunk 2]
```diff
@@ -139,14 +140,18 @@ export class MockRTCPeerConnection {
     addTrack(track: MockMediaStreamTrack) {
```
Comment (SimonBrandner) [inline spec/test-utils/webrtc.ts:140]: Return type, please

**Review by SimonBrandner — APPROVED** [review]
LGTM though CI seems unhappy

**Pull request merged**